### PR TITLE
fix(examples): resolve string formatting compiler warnings

### PR DESCRIPTION
**Pull request by douglas-healy**
### Description of the feature or fix

This change resolves compiler warnings generated from string formatting operations in certain example files.

### CODE REVIEW

File: examples/layouts/flex/lv_example_flex_1.c [hunk 1]
```diff
@@ -28,7 +28,7 @@ void lv_example_flex_1(void)
         lv_obj_set_size(obj, 100, LV_PCT(100));
 
         label = lv_label_create(obj);
-        lv_label_set_text_fmt(label, "Item: %u", i);
+        lv_label_set_text_fmt(label, "Item: %lu", i);
```
Comment (embeddedt) [inline examples/layouts/flex/lv_example_flex_1.c]: These should probably just be replaced with the `LV_PRI` macros as well; there is no way to easily get the old-style placeholders to work correctly on every platform.

Comment (douglas-healy) [reply examples/layouts/flex/lv_example_flex_1.c]: Thanks @embeddedt, I will look into that. I see some of the checks for certain platforms did fail so you are right about using the `LV_PRI` macros.

**Comment by kisvegabor** [comment]
All tests pass now. This PR is "draft" so I wonder if there is still something to so.

**Comment by douglas-healy** [comment]
> All tests pass now. This PR is "draft" so I wonder if there is still something to so.

@kisvegabor, this PR is ready for merge.

**Review by embeddedt — APPROVED** [review]

**Pull request merged**